[Bug]: Detected mismatched Protobuf Gencode/Runtime major versions

### What happened?

Upgraded from 0.2.6 to 0.2.7 and got this error.


Found while working on https://github.com/mozilla-ai/any-agent/issues/410

### Relevant log output

```shell
ImportError while loading conftest '/Users/nbrake/scm/any-agent/tests/conftest.py'.
tests/conftest.py:12: in <module>
    from any_agent.config import AgentFramework
src/any_agent/__init__.py:2: in <module>
    from .frameworks.any_agent import AgentRunError, AnyAgent
src/any_agent/frameworks/any_agent.py:14: in <module>
    from any_agent.tools.wrappers import _wrap_tools
src/any_agent/tools/__init__.py:1: in <module>
    from .a2a import a2a_tool, a2a_tool_async
src/any_agent/tools/a2a.py:17: in <module>
    from a2a.client import A2ACardResolver, A2AClient
.venv/lib/python3.13/site-packages/a2a/client/__init__.py:9: in <module>
    from a2a.client.grpc_client import A2AGrpcClient
.venv/lib/python3.13/site-packages/a2a/client/grpc_client.py:7: in <module>
    from a2a.grpc import a2a_pb2, a2a_pb2_grpc
.venv/lib/python3.13/site-packages/a2a/grpc/a2a_pb2.py:12: in <module>
    _runtime_version.ValidateProtobufRuntimeVersion(
.venv/lib/python3.13/site-packages/google/protobuf/runtime_version.py:106: in ValidateProtobufRuntimeVersion
    _ReportVersionError(
.venv/lib/python3.13/site-packages/google/protobuf/runtime_version.py:50: in _ReportVersionError
    raise VersionError(msg)
E   google.protobuf.runtime_version.VersionError: Detected mismatched Protobuf Gencode/Runtime major versions when loading a2a.proto: gencode 6.31.1 runtime 5.29.5. Same major version is required. See Protobuf version guarantees at https://protobuf.dev/support/cross-version-runtime-guarantee.
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -20,6 +20,7 @@ dependencies = [
     "grpcio>=1.60",
     "grpcio-tools>=1.60",
     "grpcio_reflection>=1.7.0",
+    "protobuf==6.31.1",
 ]
 
 classifiers = [

